src/main/java/com/google/sps/dao/GeolocationDao.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.dao;
+
+import com.google.sps.Mock;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class GeolocationDao {
```

Comment:
This is not really a DAO, you need to abstract it in a interface with getters, and you can extend it with a mock implementation. E.g. https://www.tutorialspoint.com/design_pattern/data_access_object_pattern.htm